src/v/storage/disk_log_impl.cc

```diff
@@ -1105,8 +1105,8 @@ ss::future<> disk_log_impl::do_truncate(truncate_config cfg) {
                     "User asked to truncate at:{}, with initial physical "
                     "position of:{}, but internal::offset_to_filepos_consumer "
                     "could not translate physical offsets. Log state: {}",
-                    pidx,
                     cfg,
+                    pidx,
```

Comment:
can we make this separate change, I know it is small, but still unrelated with the commit message.